koku/api/report/aws/serializers.py

```diff
@@ -93,19 +96,14 @@ class QueryParamSerializer(ParamSerializer):
     )
 
     delta = serializers.ChoiceField(choices=DELTA_CHOICES, required=False)
-
-    group_by = GroupBySerializer(required=False)
-    order_by = OrderBySerializer(required=False)
-    filter = FilterSerializer(required=False)
```

Comment:
group_by, order_by, and filter fields all got moved into the `_init_tagged_fields()` method. So, they're not attributes on the Serializer class until that method is called.

This gives us a single place where we insert our tag keys into the serializer and replaces the function of the `tag_fields` attribute.